cli/inspector_structures.rs

```diff
@@ -0,0 +1,425 @@
+use deno_core::serde_json;
```

Comment:
Missing copyright; consider renaming to `cdp.rs` and adding link to https://chromedevtools.github.io/devtools-protocol/tot/